Sources/Tabs/WalletCoordinator.swift

```diff
@@ -38,7 +38,8 @@ final class WalletCoordinator: NSObject,
                         with: root,
                         networkClient: networkClient,
                         analyticsService: analyticsCenter.analyticsService,
-                        localAuthService: DummyLocalAuthService())
+                        localAuthService: DummyLocalAuthService(),
+                        credentialIssuer: "")
```

Comment:
The requirement to pass in a `credentialIssuer` was a breaking change in the Wallet SDK:
https://github.com/govuk-one-login/mobile-wallet-ios/pull/174

This was not marked as a breaking change when it was merged and the documentation is not clear.
Hopefully @benmckillop can advise us on the correct value for this when he is back from leave,

Reply:
Resolved this now :) Will update the Wallet readme separately to instruct what to do for this value.

Apologies it wasn't marked as a breaking change - it should have been!

Reply:
Easily done 🙂 we should look at the automated `diagnose-breaking-api-changes` tool again 👀 
https://x.com/fabianfett/status/1504393754412822530?s=12